tests/cli/test_train_model.py

```diff
@@ -26,6 +27,43 @@ def test_train(monkeypatch, tmp_path, output):
     assert Path(output).is_file()
 
 
+@pytest.mark.parametrize("output", [None, "mymodel.pt"])
+def test_train_explicit_validation_test(monkeypatch, tmp_path, output):
+    """Test that training via the training cli runs without an error raise
+    also when the validation and test sets are provided explicitly."""
+    monkeypatch.chdir(tmp_path)
+    shutil.copy(RESOURCES_PATH / "qm9_reduced_100.xyz", "qm9_reduced_100.xyz")
+    shutil.copy(
+        RESOURCES_PATH / "options_explicit_valid_test.yaml",
+        "options_explicit_valid_test.yaml",
+    )
+    shutil.copy(
+        RESOURCES_PATH / "options_explicit_valid.yaml",
+        "options_explicit_valid.yaml",
+    )
+    shutil.copy(
+        RESOURCES_PATH / "options_explicit_test.yaml",
+        "options_explicit_test.yaml",
+    )
+
+    structures = ase.io.read("qm9_reduced_100.xyz", ":")
+    ase.io.write("train.xyz", structures[:50])
+    ase.io.write("valid.xyz", structures[50:80])
+    ase.io.write("test.xyz", structures[80:])
+
+    command = ["metatensor-models", "train", "options_explicit_valid_test.yaml"]
+    command = ["metatensor-models", "train", "options_explicit_valid.yaml"]
+    command = ["metatensor-models", "train", "options_explicit_test.yaml"]
```

Comment:
I would rather like this as a `pytest.mark.parametrize`. These jobs creates files and this may interfere with each other.